x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/details/policy_response.tsx

```diff
@@ -51,7 +50,7 @@ const ResponseActions = memo(
     actions,
     actionStatus,
   }: {
-    actions: ImmutableArray<keyof HostPolicyResponseActions>;
+    actions: Immutable<Array<keyof HostPolicyResponseActions>>;
```

Comment:
Cool. So I assume we should try to only use `Immutable<>` and not its sub-types, right?

Reply:
sorry, let me justify myself

my goal is to encapsulate the code behind `Immutable` in case it changes (e.g. ts actually provides it.)